earth_enterprise/SConstruct

```diff
@@ -158,7 +171,7 @@ stage_tutorial = env.PhonyTargets(**stage_tutorial_cmds)
 env.Depends(stage_tutorial, stage_data)
 
 package_staged_cmds = {'package_install': [
-    'cd rpms; ./gradlew osPackage'
+    "cd rpms; ./gradlew osPackage " + CommandLineParameter().option
```

Comment:
That was minor comment, but you'd need to replace `CommandLineParameter().option` with `package_command_arguments` here.